app/views/idv/questions/index.html.slim

```diff
@@ -0,0 +1,4 @@
+- if @question.choices.nil?
+  = render 'idv/questions/text'
+- else
+  = render 'idv/questions/multiple_choice'
```

Comment:
good point. There seems to be different opinions on how/where to do views partials logic. I imagined that since both views actually share a lot of duplicate markup, it would make sense to refactor this into a better partial, which would mean keeping the logic in the view.

Is this a blocker for you? This and other UX changes will undoubtedly come up in a future PR.